gma/integration_test/src/integration_test.cc

```diff
@@ -768,6 +768,9 @@ TEST_F(FirebaseGmaTest, TestRequestConfigurationSetGet) {
 // ensure that we can load them before diving into the interactive tests.
 TEST_F(FirebaseGmaTest, TestAdViewLoadAd) {
   SKIP_TEST_ON_DESKTOP;
+  SKIP_TEST_ON_SIMULATOR;
+
+  FLAKY_TEST_SECTION_BEGIN();
 
   const firebase::gma::AdSize banner_ad_size(kBannerWidth, kBannerHeight);
   firebase::gma::AdView* ad_view = new firebase::gma::AdView();
```

Comment:
Does AdView let you call Load again if the first load fails? If so, I think it would be better to restrict the flaky section to just the LoadAd / WaitForCompletion. I know we can't do that with Interstitial and Rewarded, but with Banners we should be able to, no?

Reply:
Done.